DevGuide.md

```diff
@@ -303,11 +303,10 @@ Building the GhidraDev plugin for Eclipse requires the CDT and PyDev plugins for
 Download `cdt-8.6.0.zip` from The Eclipse Foundation, and place it in a directory named:
 `ghidra.bin/GhidraBuild/EclipsePlugins/GhidraDev/buildDependencies/`. Note that
 `ghidra.bin` must be a sibling of `ghidra`.
-To respect the Eclipse Project's resources, you may need to download the file using a browser, or at the very least, locate a suitable mirror on your own:
 
 ```bash
 cd ~/Downloads   # Or wherever
-curl -OL https://$CHOOSE_YOUR_MIRROR/pub/eclipse/tools/cdt/releases/8.6/cdt-8.6.0.zip
+curl -OL 'http://www.eclipse.org/downloads/download.php?r=1&protocol=https&file=/tools/cdt/releases/8.6/cdt-8.6.0.zip'
```

Comment:
Awesome. I was trying to figure this out, and was also irritated that it interrupted the copy-paste method. However, given that it doesn't obey `protocol=https`, I think we should include an explicit checksum step. I downloaded a copy manually over https and recorded my checksum in the comment. I couldn't find an official checksum....

```suggestion
curl -OL 'http://www.eclipse.org/downloads/download.php?r=1&protocol=https&file=/tools/cdt/releases/8.6/cdt-8.6.0.zip'
sha256sum cdt-8.6.0.zip
# 81b7d19d57c4a3009f4761699a72e8d642b5e1d9251d2bb98df438b1e28f8ba9
```

Reply:
That makes a whole lot of sense, yeah.  Fortunately it looks like [there's a good way to automate checksum verification](https://wiki.eclipse.org/CBI/How_to_check_integrity_of_downloads_from_the_Eclipse_Foundation#Improving_automation), plus it apparently serves up checksums over HTTPS, which addresses my next concern.

I'll do a tiny bit of prototyping here and see about adding a copy-and-paste friendly checksum verification line; if I work it out I'll pop another commit into the PR.